stratisd_cert.py

```diff
@@ -1001,7 +1012,7 @@ def test_get_report(self):
         """
         Test getting a valid and invalid report.
         """
-        (result, return_code, _) = StratisDbus.get_report("stopped_pools")
+        (result, return_code, _) = StratisDbus.get_report("errored_pool_report")
```

Comment:
This is just a rebase problem....

Reply:
Actually, no; this is the commit `Revert "stratisd: change test_get_report valid test to stopped_pools"` in this PR's branch, which I would need, to ensure that this test passes.  Is it okay if I add this commit?

Reply:
@bgurney-rh Go ahead.